js/src/spbc-table.js

```diff
@@ -694,111 +694,139 @@ function spbcShowHideRows(blockId, columnKey) {/* eslint-disable-line no-unused-
 }
 
 /**
- * Update estimated time
+ * Starts an interval to update the analysis log table and stats every 6 seconds.
  */
-function spbcUpdateEstimatedTime() {
-    spbcUpdateTables();
+function spbcEATUpdateIntervalStart() {
     setInterval(() => {
-        spbcUpdateTables();
+        // Check if the analysis log tab is present
+        if (document.querySelector('div[refresh_control_tab="analysis_log"]') !== null) {
+            // Add a spinner to the analysis log tab header
+            spbcEATAddSpinner();
+            // Update the analysis log table
+            spbcUpdateTableEAT();
+            // Update the analysis log stats
+            spbcUpdateStatsEAT();
+        }
     }, 60000);
 }
 
 /**
- * Update tables
- * @param {obj} tab
- * @param {obj} params
+ * Adds a spinner to the analysis log tab header if it doesn't already exist.
+ */
+function spbcEATAddSpinner() {
+    if (document.getElementById('analysis_log_counter_spinner') === null) {
+        const refreshTab = document.querySelector('h3[aria-controls="spbc_scan_accordion_tab_analysis_log"]');
+        if (refreshTab !== null) {
+            const spinner = document.createElement('img');
+            spinner.class = 'spbc_preloader';
+            spinner.alt = 'Wait for refresh..';
+            spinner.src = '../../wp-content/plugins/security-malware-firewall/images/preloader2.gif';
```

Comment:
The 'src' attribute must be absolute URL path. Mabe, make this through JS localization.